symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -652,6 +655,33 @@ public FollowingListResponse listUsersFollowing(@Nonnull Long userId, @Nonnull C
     return new PaginatedService<>(api, pagination.getChunkSize(), pagination.getTotalSize()).stream();
   }
 
+  /**
+   * Return a {@link V2UserDetail} with created user details.
+   *
+   * @param payload User's details to create.
+   * @return a {@link V2UserDetail} with created user details.
+   * @see <a href="https://developers.symphony.com/restapi/reference#create-user-v2">Create User v2</a>
+   */
+  @API(status = API.Status.STABLE)
+  public V2UserDetail create(@Nonnull V2UserCreate payload) {
```

Comment:
Is it allowed to pass an OpenAPI generated model in arguments here ?  (like https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/bafc4cf0c3e3cdf8cb934a169952fe3876234a89/symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java#L541 )

Reply:
It is manually added but it is useless here as it is put at class level (line 70)